src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsFormHeader.tag

```diff
@@ -6,9 +6,9 @@
 <%@ attribute name="fsForm" type="teammates.ui.template.FeedbackSessionsForm" required="true"%>
 
 <c:if test="${fsForm.feedbackSessionTypeEditable}">
-  <div class="row">
+  <div class="row" style="margin-right: 6px;">
```

Comment:
u can try `style="float:right"`

Reply:
@wenmogu Thanks for the suggestion! This works well for the desktop version, i.e. when the page is full-screen. However, for smaller screens it's a different story. For easy comparison, I have taken screen shots for how the UI appears on smaller screens:

**Original**
<img width="566" alt="current" src="https://user-images.githubusercontent.com/24493463/37567780-a515552a-2b06-11e8-98fc-025f9d1e2cd8.png">

**My hardcoded fix**
<img width="554" alt="my" src="https://user-images.githubusercontent.com/24493463/37567782-abad7674-2b06-11e8-9257-6ec6e69e7bbe.png">

**Your proposal**
<img width="561" alt="wx" src="https://user-images.githubusercontent.com/24493463/37567784-b0ce0574-2b06-11e8-9bcd-40f17c6a9c52.png">

Although it is probably more likely that TEAMMATES is used on a laptop/ desktop rather than devices with smaller screen width, I am not sure if we are okay with this look for smaller screens (if I follow your suggestion).

Reply:
That said, I am actually more in favour of your suggestion (than my hardcoded fix) because it works better on the instructor help page as well (where the question form is smaller than it actually is). So I will adopt your switch (at least for the time being).